prometheus/base/alerting/prometheus.pod_alerts.yaml

```diff
@@ -25,3 +25,14 @@ spec:
       annotations:
         summary: "Pod {{ $labels.pod }} is crash looping"
         description: "Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in waiting state (reason: 'CrashLoopBackOff') for more than 15 minutes."
+    - alert: PodsNotReady
+      expr: |
+        sum by (namespace, pod)
+        (max by (namespace, pod) (kube_pod_status_phase{phase=~"Pending|Unknown|Failed", namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"}
```

Comment:
I think max by (namespace, pod) is not required here

Reply:
removed